programmer-humour/script.js

```diff
@@ -0,0 +1,37 @@
+const api = "https://xkcd.now.sh/?comic=latest"
+let lastComic
+
+document.addEventListener('DOMContentLoaded', function () {
+  fetchComic(api);
+});
+
+function fetchComic(api) {
+  fetch(api)
+    .then((response) => response.json())
+    .then((data) => lastComic = data)
```

Comment:
Do you need to save `lastComic` as a variable at the top of the file?

Could you just pass it into the next `then` callback instead?

Reply:
Variable deleted. Thanks.